### PR TITLE
5 Set up Project

### LINKED ISSUE #3: Host can add space
As a host,
So that I can promote my space,
I want to create a new space

### LINKED ISSUE #5: Project Set up
As a Developer, 
I want to have a functioning project

### PR DESCRIPTION
**Pull request by ccfz**
closes #5

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require 'data_mapper'
+require './app/app.rb'
+
+# if ENV['RACK_ENV'] != 'production'
```
Comment (robota-x) [inline Rakefile]: remove the commented code

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+require 'database_cleaner'
+require 'sinatra/flash'
+
+
+Capybara.app = Bnb
+
+RSpec.configure do |config|
+  config.include Capybara::DSL
+
+#     config.before(:suite) do
```
Comment (lorenzoturrino) [inline spec/spec_helper.rb]: more commented code to remove

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+require 'simplecov'
+
+
+SimpleCov.formatters = [
+  SimpleCov::Formatter::HTMLFormatter,
+  Coveralls::SimpleCov::Formatter
+]
+SimpleCov.start
+
+require File.join(File.dirname(__FILE__), '..', 'app/app.rb')
```
Comment (lorenzoturrino) [inline spec/spec_helper.rb]: should we leave this like that?

File: Gemfile [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+gem 'sinatra-partial'
+gem 'bcrypt'
+gem 'sinatra-flash'
+gem 'pry'
+
+
+group :test do
+  gem 'rspec'
+  gem 'cucumber'
+  gem 'rake'
```
Comment (robota-x) [inline Gemfile]: rake is needed in the main group not test

**Comment by robota-x** [comment]
:shipit:

**Pull request merged**